internal/platform/types_test.go

```diff
@@ -62,3 +62,33 @@ func TestOpenShiftVersion_String(t *testing.T) {
 	assert.Equal(t, "OpenShiftVersion [Version: 1.1.1+]",
 		info.String(), "OpenShiftVersion String() yields malformed result of %s", info.String())
 }
+
+func TestVersionComarsion(t *testing.T) {
```

Comment:
small change - typo, Comparison missing the 'p'

Reply:
corrected.